test/types/temp_comm.jl

```diff
@@ -0,0 +1,86 @@
+using COBREXA.Reconstruction
```

Comment:
pls wrap this into a testset, also if possible keep all the imports in `runtests.jl`

Reply:
will do

Reply:
not resolved yet

Reply:
oh,   I forgot about it!